templates/prometheus/030-Deployment-prometheus.yaml

```diff
@@ -12,7 +12,7 @@ spec:
         name: prometheus
     spec:
       containers:
-      - image: prom/prometheus:v2.0.0
+      - image: prom/prometheus:latest
```

Comment:
This should be configurable. Replace with ${PROMETHEUS_IMAGE} and change Makefile to to a substituttion